docker/Dockerfile

```diff
@@ -46,15 +46,35 @@ WORKDIR /home/pyth
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs \
   | sh -s -- -y 
 
-COPY --chown=pyth:pyth . pyth-client/
+ENV PATH=${PATH}:/home/pyth/.cargo/bin
 
-# Build off-chain binaries.
-RUN cd pyth-client && ./scripts/build.sh
+COPY --chown=pyth:pyth ./scripts/get-bpf-tools.sh pyth-client/scripts/
+RUN ls pyth-client/scripts && ./pyth-client/scripts/get-bpf-tools.sh
```

Comment:
Why `ls`

Reply:
Docker debugging, forgot to remove this one, sorry!